### ISSUE TITLE
[Doc] TypeError: check_type() missing 1 required positional argument: 'expected_type'

### ISSUE THREAD

**Issue by ayaka14732** [agronholm/typeguard#269]
**Describe the bug**

https://typeguard.readthedocs.io/en/latest/userguide.html#checking-types-directly

```python
from typeguard import check_type

# Raises TypeError if there's a problem
check_type([1234], List[int])
```

**To Reproduce**

Run the above code. Get error:

```
TypeError: check_type() missing 1 required positional argument: 'expected_type'
```

**Suggested changes**

```python
from typeguard import check_type

a = [1234]
# Raises TypeError if there's a problem
check_type('a', a, List[int])
```

**Comment by agronholm** [comment]
This is an artifact of the published documentation being more up to date than the latest release. I have been busy with other projects for some time but I will endeavor to make a new major release of typeguard soon. That will fix the discrepancy you're seeing.

**Comment by fpgmaas** [comment]
For others stumbling onto this issue, as a current workaround you can install the `master` branch:

```
poetry add git+https://github.com/agronholm/typeguard.git#master
pip install git+https://github.com/AGeekInside/fastapi-jinja.git#master
```

**Comment by agronholm** [comment]
I've made the stable branch in the docs the default. That branch correctly specifies `check_type()` to have 3 parameters as in the latest release.

**Comment by agronholm** [comment]
I've released v3.0.0 so this shouldn't be an issue anymore.
